lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module Capybara
+        # Checks for if there is a more specific matcher offered by Capybara.
+        #
+        # @example
+        #
+        #   # bad
+        # expect(page).to have_css('button')
```

Comment:
I believe it's due to the lack of indentation here.

Reply:
Thank you very much. Fixed the indentation!